content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,105 @@ Ogone
 `Ogone <https://www.ingenico.com/>`_, also known as **Ingenico Payment Services** is a France-based
 company that provides the technology involved in secure electronic transactions.
 
-Configuration on Odoo
-=====================
+Configuration
+=============
 
 .. seealso::
    - :ref:`payment_acquirers/add_new`
+   - `Ogone's documentation <https://epayments-support.ingenico.com/get-started/>`_.
 
-Credentials tab
----------------
+Settings in Ogone
+-----------------
 
-Odoo needs your **API Credentials** to connect with your Ogone account, which comprise:
+Create an API user
+~~~~~~~~~~~~~~~~~~
 
-- **PSPID**: The ID solely used to identify the account with Ogone. You chose it when you opened
-  your account.
-- :ref:`API User ID <ogone/api_user>`: The ID solely used to identify the user with Ogone.
-- :ref:`API User Password <ogone/api_user>`: Value used to identify the user with Ogone.
-- :ref:`SHA Key IN <ogone/sha_key_in>`: Key used in the signature Odoo send to Ogone.
-- :ref:`SHA Key OUT <ogone/sha_key_out>`: Key used in the signature Ogone send to Odoo.
+Log into your Ogone account and head to the :guilabel:`Configuration` tab.
 
-You can copy your credentials from your Ogone account, and paste them in the related fields under
-the **Credentials** tab.
+You need to create an **API user** to be used in the creation of transactions from Odoo. While you
+can use your main account to do so, using an **API user** ensures that if the credentials used in
+Odoo are leaked, no access to your Ogone configuration is possible. Additionally, passwords for
+**API users** do not need to be updated regularly, unlike normal users.
 
-.. _ogone/api_user:
+To create an **API user**, go to :menuselection:`Configuration --> Users` and click on
+:guilabel:`New User`. The following fields must be configured:
 
-API User ID and Password
-~~~~~~~~~~~~~~~~~~~~~~~~
+.. _ogone/ogone:
 
-If you already created a user and have both its ID and password, just copy them. You can also
-generate a new password from :menuselection:`Configuration --> Users --> Your chosen user --> change
-password`.
+- :guilabel:`UserID`: you can choose anything you want, but the convention is to name it `OOAPI`.
```

Comment:
There is no such convention, and I'm not sure if suggesting a cryptic name (is it O o 0) is a good idea. Wouldn't it be better to simply show an example?
```suggestion
- :guilabel:`UserID`: you can choose anything you want. For example: `APIUSER`.
```